### DISCUSSION TITLE
The problems of loading big md files

### DISCUSSION THREAD

**Discussion by viifirm** [docsifyjs/docsify#2400] [Q&A] [answered]
Dear Docsify teams and contributors, 
Appreciate your effort to bringing such a great system to the world. docsify is almost my favorite infraustructure. I have been building a few cool website using it. But one single problem I found is when it loading big md file will be very slow(>500k or >1MB), sometime the progress bar will runs a few time. without split the file into pieces, what would be a better solution for that? such as different rendering policy when loading big file, or a plugin to load the big file by sections. any good idea if I try to develop a plugin of that? Thanks.

**Comment by sy-records** [comment, marked as answer]
You may need to look first to see if it's slow in getting the file content or slow in parsing the html?

**Reply by viifirm** [reply]
I checked local deployment, i think 90% of problem is loading the content, so its more on internet problem. thanks. by the way, Can I change program behaviour for some md file? such as default setting is not loading side bar, but for some file requires to load _sidebar.md. is that possible? thanks again

**Reply by sy-records** [reply]
You may want the sidebar to update after navigation to reflect the current directory. This can be done by adding a _sidebar.md file to each folder.

**Reply by viifirm** [reply]
Thanks a lot
